regparser/layer/formatting.py

```diff
@@ -236,7 +236,7 @@ def match_data(self, match):
 
 
 class Footnotes(PlaintextFormatData):
-    """E.g.     [^4](Contents of footnote with escaped parens: \(\)s)"""
+    r"""E.g.     [^4](Contents of footnote with escaped parens: \(\)s)"""
```

Comment:
I might not be understanding this correctly, but I would consider trying something like:

```
"""
Handles footnotes. A variable with text that would match looks something like this::

    fn = r"[^4](Contents of footnote with escaped parens: \(\)s)"
"""
```